packages/launchpad/src/store/app.ts

```diff
@@ -0,0 +1,98 @@
+import { reactive, readonly, inject, App } from 'vue'
+
+interface State {
+  projectTitle: string
+  title: string
+  description: string
+  nextAction: () => void
+  backAction: () => void
+  alternativeAction?: () => void
+  steps: {
+      component?: boolean
```

Comment:
Should these be optional? Maybe just `true` and `false`? I think we should only make things optional when `undefined` has a different meaning to `false`.